lib/measures/tbd/measure.rb

```diff
@@ -248,7 +250,7 @@ def arguments(model = nil)
     alter_model = OpenStudio::Measure::OSArgument.makeBoolArgument("alter_model", true, false)
     alter_model.setDisplayName("Alter OpenStudio model (Apply Measures Now)")
     alter_model.setDescription("If checked under Apply Measures Now, TBD will irrevocably alter the user's OpenStudio model.")
-    alter_model.setDefaultValue(false)
+    alter_model.setDefaultValue(true)
```

Comment:
This is an oversight. Very handy if `false` for _Apply Measures Now_ iterative exploration, but if left `false` in normal simulation situations, E+ results are unaffected - not a desirable outcome! So the best is to leave this to `true` by default.